src/vuestic-theme/vuestic-components/vuestic-tabs/VaTabs.demo.vue

```diff
@@ -0,0 +1,59 @@
+<template>
```

Comment:
On removal  selection moves to left. Should stay on same place.
![image](https://user-images.githubusercontent.com/5394573/54205490-80c46700-44e7-11e9-9d8b-ae151874bcf3.png)